layouts/partials/banner.md

```diff
@@ -1,9 +1,9 @@
 {{ if .Params.show_banner -}}
 <div class="o-banner">
 
-The OpenTelemetry tracing specification is now 1.0!
+OpenTelemetry Metrics Release Candidates are now available!
 [Learn more][]
 
-[Learn more]: https://medium.com/opentelemetry/opentelemetry-specification-v1-0-0-tracing-edition-72dd08936978
+[Learn more]: https://opentelemetry.io/blog/2022/metrics-announcement/
```

Comment:
Should be a local path -- we want to avoid "external" links into the website itself.

Reply:
Hm, I tried using the {{< relref >}} directive and the build failed. What's the right way to do relative links from this component?

Reply:
No need for relref (and I've been thinking of removing relref usage from the site pages, I can explain why at the next Comms meeting), just use the path `/blog/2022/metrics-announcement/`.